src/main/java/helloworld/s3_lambda_sqs/SQSMessageSender.java

```diff
@@ -0,0 +1,51 @@
+package helloworld.s3_lambda_sqs;
+
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.AmazonSQSClientBuilder;
+import com.amazonaws.services.sqs.model.AmazonSQSException;
+import com.amazonaws.services.sqs.model.CreateQueueResult;
+import com.amazonaws.services.sqs.model.SendMessageBatchRequest;
+import com.amazonaws.services.sqs.model.SendMessageBatchRequestEntry;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.logging.Handler;
+
+import static helloworld.constant.Constants.MESSAGE;
+import static helloworld.constant.Constants.MESSAGE_SENT;
+import static helloworld.constant.Constants.QUEUE_NAME;
+import static helloworld.constant.Constants.WITH_DELAY_SECONDS;
+
+public class SQSMessageSender {
+    Gson gson = new GsonBuilder().setPrettyPrinting().create();
+    private static final Logger logger = LoggerFactory.getLogger(Handler.class);
+
+    public static void main(String[] args) {
+        final AmazonSQS sqs = AmazonSQSClientBuilder.defaultClient();
+
+        // Create queue if doesn't exists
+        try {
+            CreateQueueResult create_result = sqs.createQueue(QUEUE_NAME);
```

Comment:
I'd suggest keeping the camelCase convention :)
```suggestion
            CreateQueueResult createResult = sqs.createQueue(QUEUE_NAME);
```